distribution/ddf-common/src/main/resources/etc/custom.properties

```diff
@@ -46,59 +46,218 @@ org.apache.karaf.security.providers=sun.security.provider.Sun,org.bouncycastle.j
+    org.apache.xerces.jaxp.datatype;uses:=javax.xml.datatype;version=2.11.0, \
+    org.apache.xerces.jaxp.validation;uses:=\"javax.xml.transform,javax.xml.validation,org.apache.xerces.xni.grammars,org.w3c.dom.ls,org.xml.sax\";version=2.11.0, \
+    org.apache.xerces.parsers;uses:=\"org.apache.xerces.dom,org.apache.xerces.dom3.as,org.apache.xerces.impl,org.apache.xerces.impl.dtd,org.apache.xerces.impl.dv,org.apache.xerces.impl.validation,org.apache.xerces.impl.xs,org.apache.xerces.util,org.apache.xerces.xinclude,org.apache.xerces.xni,org.apache.xerces.xni.grammars,org.apache.xerces.xni.parser,org.apache.xerces.xs,org.w3c.dom,org.w3c.dom.ls,org.xml.sax,org.xml.sax.ext\";version=2.11.0, \
+    org.apache.xerces.stax;uses:=\"javax.xml.namespace,javax.xml.stream,javax.xml.stream.events\";version=2.11.0, \
+    org.apache.xerces.stax.events;uses:=\"javax.xml.namespace,javax.xml.stream,javax.xml.stream.events\";version=2.11.0, \
+    org.apache.xerces.util;uses:=\"javax.xml.namespace,javax.xml.stream,org.apache.xerces.dom,org.apache.xerces.impl,org.apache.xerces.xni,org.apache.xerces.xni.grammars,org.apache.xerces.xni.parser,org.w3c.dom,org.w3c.dom.ls,org.xml.sax,org.xml.sax.ext\";version=2.11.0, \
+    org.apache.xerces.xinclude;uses:=\"org.apache.xerces.impl,org.apache.xerces.util,org.apache.xerces.xni,org.apache.xerces.xni.parser,org.apache.xerces.xpointer\";version=2.11.0, \
+    org.apache.xerces.xni;uses:=org.apache.xerces.xni.parser;version=2.11.0, \
+    org.apache.xerces.xni.grammars;uses:=\"org.apache.xerces.xni,org.apache.xerces.xni.parser,org.apache.xerces.xs\";version=2.11.0, \
+    org.apache.xerces.xni.parser;uses:=org.apache.xerces.xni;version=2.11.0, \
+    org.apache.xerces.xpointer;uses:=\"org.apache.xerces.impl,org.apache.xerces.util,org.apache.xerces.xinclude,org.apache.xerces.xni,org.apache.xerces.xni.parser\";version=2.11.0, \
+    org.apache.xerces.xs;uses:=\"org.apache.xerces.xs.datatypes,org.w3c.dom,org.w3c.dom.ls\";version=2.11.0, \
+    org.apache.xerces.xs.datatypes;uses:=\"javax.xml.datatype,javax.xml.namespace,org.apache.xerces.xni,org.apache.xerces.xs\";version=2.11.0, \
+    org.apache.xml.serialize;uses:=\"org.apache.xerces.dom,org.apache.xerces.util,org.w3c.dom,org.w3c.dom.ls,org.xml.sax,org.xml.sax.ext\";version=2.11.0, \
+    org.apache.karaf.diagnostic.core;uses:=org.osgi.framework;version=4.0.4, \
+    org.apache.karaf.diagnostic.core.common;uses:=org.apache.karaf.diagnostic.core;version=4.0.4, \
+    org.apache.karaf.jaas.boot;uses:=\"javax.security.auth,javax.security.auth.callback,javax.security.auth.login,javax.security.auth.spi,org.osgi.framework\";version=4.0.4, \
+    org.apache.karaf.jaas.boot.principal;uses:=javax.security.auth;version=4.0.4, \
+    com.sun.crypto.provider, \
+    com.sun.net.ssl, \
+    com.sun.net.ssl.internal.ssl
+
+#  sun.misc, \
```

Comment:
It should be safe to remove these - if they were required by a bundle then I should have seen failures by now.